### ISSUE TITLE
discitis

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#7177]
**Preferred term label:**
discitis

**Synonyms**
inflammation of intervertebral disk
intervertebral disk inflammation

**Definition (free text, please give PubMed ID)**
Inflammation of an intervertebral disc or disk space which may lead to disk erosion. Until recently, discitis has been defined as a nonbacterial inflammation and has been attributed to aseptic processes (e.g., chemical reaction to an injected substance). However, recent studies provide evidence that infection may be the initial cause, but perhaps not the promoter, of most cases of discitis. Discitis has been diagnosed in patients following discography, myelography, lumbar puncture, paravertebral injection, and obstetrical epidural anesthesia. Discitis following chemonucleolysis (especially with chymopapain) is attributed to chemical reaction by some and to introduction of microorganisms by others.
MESH:D015299

**Parent term (use hpo.jax.org/app)**
Abnormality of the intervertebral disk HP:0005108

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
infectious dicitis
see https://github.com/monarch-initiative/mondo/issues/3852

**Comment by pnrobinson** [comment]
added as HP_0025679
